packages/beacon-node/src/api/impl/beacon/blocks/utils.ts

```diff
@@ -65,6 +65,13 @@ async function resolveBlockIdOrNull(
     };
   }
 
+  if (blockId === "justified") {
+    return {
+      block: await db.blockArchive.get(forkChoice.getJustifiedBlock().slot),
```

Comment:
Non finalized blocks are not in the archive, but in the block hot db. blockArchive will most likely return null for this query. Instead grab by root from the hot db.

Reply:
Just pushed a fix for this, thanks @nazarhussain for helping me to validate the solution.